.github/labeller.yml

```diff
@@ -0,0 +1,32 @@
+# Auto Label Git Config
+# 'label_name'
+# -path_of_diff/where/code_lives/to_base_label upon
+
+# Any changes to any file in the .github/workflow directory
+'Github Actions':
+  - .github/workflows/*
+
+# Any changes to any file in the dockerfiles and docker-compose.yml
+'Docker':
+  - '**/Dockerfile'
+  - docker-compose.yml
+
+# Any changes to any file or subdirectories in the Frontend directory
+'Frontend':
+  - /frontend/**/*
+
+# Any changes to any file or subdirectories in the Backend directory
+'Backend':
+  - /backend/**/*
+
+# Any changes to any environment variables
+'Environment Variables':
+  - '/**/*.env*/'
+
+# Any changes to any swagger documents
+'Swagger':
+  - /backend/src/docs/*.yml
+
+# Any changes to any swagger configurations
+'Swagger Configurations':
+  -/backend/src/config/*.ts
```

Comment:
```suggestion
  - /backend/src/config/*.ts
```

Reply:
Good eyes!
Shouldn't we expand the linter and formatter to be running on the root dir files too?

Reply:
More trouble than its worth
Your root files will seldomly change after setup (besides the README), and your linter wouldn't catch this example anyways.

Reply:
All done!